external/vis.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google LLC
+# Modifications copyright 2020 Aalto University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+import numpy as np
+
+
+def draw_layout_direct(elements: iter, size=None, bgcolor=(255, 255, 255)):
```

Comment:
Used this in my own project as well to generate layouts, find it useful to debug results